blockonomics-woocommerce.php

```diff
@@ -531,10 +530,10 @@ function blockonomics_update_db_check() {
 
     $installed_ver = get_site_option( 'blockonomics_db_version' );
     if ( $installed_ver != $blockonomics_db_version ) {
-        // Example function to demonstrate table changes between upgrade versions
-        // if ($installed_ver < 1.0) {
-        //     $wpdb->query("ALTER TABLE $table_name DROP transaction;");
-        // }
+        $table_name = $wpdb->prefix . 'blockonomics_orders';
+        if ($blockonomics_db_version > 1.0) {
```

Comment:
This code will run again incorrectly when `if ($blockonomics_db_version > 1.1)` is added and the user upgrades from db_version 1.1->1.2.
We should use `if ($installed_ver < 1.1)` as we want to run the upgrades between the installed version and the new db_version. This also makes the code easier to read as the if statement references the matching db_version

Reply:
Ah, that makes more sense. Thanks for the clarification! I've pushed the change